spec/slather/coverage_service/coveralls_spec.rb

```diff
@@ -13,13 +13,6 @@
     end
   end
 
-  describe "#travis_job_id" do
-    it "should return the TRAVIS_JOB_ID environment variable" do
-      ENV['TRAVIS_JOB_ID'] = "9182"
-      expect(fixtures_project.send(:travis_job_id)).to eq("9182")
```

Comment:
You could probably re-enable this, but just make sure to *restore* the previous value. So that when coveralls runs has the correct job ID.


```ruby
actual_travis_job_id = ENV['TRAVIS_JOB_ID']
ENV['TRAVIS_JOB_ID'] = "9182"
expect(fixtures_project.send(:travis_job_id)).to eq("9182")
ENV['TRAVIS_JOB_ID'] = actual_travis_job_id
```